Makefile

```diff
@@ -75,8 +75,10 @@ setup:
 	$(MAKE) install
 	$(MAKE) -C assets $@
 
+export ENDPOINT_HOST ?=
+
+build-release: ENDPOINT_HOST=pay-$(STG).apps.y.cld.gov.au
```

Comment:
if we can export `ENDPOINT_HOST` as part of the feature branch detection it makes this target useful outside of CI and across cld environments.

line 40 could look like

```
export ENDPOINT_HOST ?= pay-$(STG).$(CLD_HOST)
```

This would pick up the correct cloud environment, and not override user-supplied environment variable; for instance in prod or if we need to locally work with a release.

Reply:
@dagoof that would work much better thanks.